processor/sourcemap/payload.go

```diff
@@ -35,7 +36,7 @@ func mappings(pa *payload) (time.Time, []m.DocMapping) {
 				return common.MapStr{
 					"bundle_filepath": pa.BundleFilepath,
 					"service":         common.MapStr{"name": pa.ServiceName, "version": pa.ServiceVersion},
-					"sourcemap":       pa.Sourcemap,
+					"sourcemap":       base64.StdEncoding.EncodeToString(pa.Sourcemap),
```

Comment:
@roncohen what do you think of storing a base64 encoded string instead?
This way we still have the `[]byte` for validating the sourcemap when uploading it and we simply can Decoded the base64 string when applying the sourcemapping for requests.

Reply:
that's basically what the `binary` field type does AFAIK. Wouldn't it be better in the long term to add support for this field to beats instead of building our own thing on top of a field that isn't meant for binary data storage?

I haven't looked at how ES implements the `binary` field, but I could imagine that it stores it in a less wasteful format than base64 (33% overhead in the best case!), and only uses base64 for transmission.

Reply:
I wasn't sure what the kibana index pattern generator was so I attempted to add binary field support in elastic/beats#5974.

Reply:
nice @graphaelli I am working on the same thing atm ;) 
I am also working on adding support for source_filtering as I have figured out now how to exclude information from being shown in discover.

Reply:
Awesome stuff @simitt & @graphaelli!!

Reply:
@beniwohli the binary type accepts a base64 encoded string, that's why I changed it to this here, sorry for not being clear enough that this also includes adding binary support for creating the Kibana index pattern and ES template out of fields.yml. 
see https://www.elastic.co/guide/en/elasticsearch/reference/current/binary.html#binary

Reply:
ah, I assumed that the go elasticsearch library would handle that transparently

Reply:
Adding source filteringin beats:  https://github.com/elastic/beats/pull/5975